HIPs/incubation/waterfall.md

```diff
@@ -0,0 +1,236 @@
+---
+layout: default
+title: Waterfall Network
+parent: Incubation
+grand_parent: Hyperledger Improvement Proposals
+---
+
+# HIP Identifier
+Hyperledger Waterfall Network v1.0
+
+# Sponsor(s)
+
+- Ron Resnick [<ron@bluewave.global>](mailto:ron@bluewave.global)
+- Richard Wang [<rwang@draperdragon.com>](mailto:rwang@draperdragon.com)
+- Jeff McDonald [<jeff@waterfall.network>](mailto:jeff@waterfall.network)
+- Michael Terpin [<michael@transformventures.io>](mailto:michael@transformventures.io)
+- Dmytro Dmytryshyn, ScD [<dmitrishin@op.edu.ua>](mailto:dmitrishin@op.edu.ua)
+- Sergii Grybniak, PhD [<sergii@bluewave.globall>](mailto:sergii@bluewave.global)
```

Comment:
sergii@bluewave.globall -> sergii@bluewave.global

Reply:
Thank you. to be fixed